wfdb/plot/plot.py

```diff
@@ -8,10 +9,10 @@
 from ..io.annotation import Annotation
 
 
-def plot_items(signal=None, ann_samp=None, ann_sym=None, fs=None,
+def plot_items(signal=None, ann_samp=None, ann_sym=None, ann_aux = None, fs=None,
```

Comment:
What does `ann_aux` do? Where is the docstring for that one?

Reply:
Also you are changing styles throughout, I would remove the spaces around the equal sign to keep it all consistent.